manifests/init.pp

```diff
@@ -28,7 +29,8 @@
   }
 
   class { "rpmbuilder::packages::essential":
-    epel  => $add_epel,
+    epel         => $add_epel,
+    mock_version => $mock_version,
```

Comment:
Maintaining defaults at both levels feels gross to me. I'm not blocking on it, I just feel like the packages class itself has a sane default and should own the default, whatever that means.